.github/workflows/test.yml

```diff
@@ -69,9 +69,9 @@ jobs:
     - name: Build sasdata docs (Ubuntu only)
       if: matrix.os == 'ubuntu-latest'
       run: |
-        mkdir -p build/lib
-        sphinx-apidoc -o docs/source/dev/generated/ -d 8 -H SasData build/lib
-        sphinx-build -M html docs/source/ docs/build/
+        cd docs
```

Comment:
The three lines can be replaced with `make -C docs html` — let make deal with the working directory for us.